servico.html

```diff
@@ -1,66 +1,71 @@
     <title>Lista</title>
     <style>
-
-
-
+      .gallery {
+        display: grid;
+        grid-template-columns: repeat(4, 1fr); /* Creates a 4 layout */
+        justify-content: space-between;
+        grid-auto-rows: minmax(100px, auto); /* Sets a minimum row height */
+        gap: 1rem; /* Adjust the gap to your preference */
+      }
+
+      @media (max-width: 1024px) {
         .gallery {
-            display: grid;
-            grid-template-columns: repeat(3, 1fr); /* Creates a three-column layout */
-            justify-content: space-between;
-            grid-auto-rows: minmax(100px, auto); /* Sets a minimum row height */
-            gap: 1rem; /* Adjust the gap to your preference */
-        }
-
-        @media (max-width: 768px) {
-            .gallery {
-                grid-template-columns: repeat(2, 1fr); /* Two columns for smaller screens */
-            }
-        }
-
-
-        .gallery-item {
-            display: flex;
-            flex-direction: column;
-            margin: 5px;
-            justify-content: flex-end; /* Aligns the description to the bottom */
-        }
-
-
-        .gallery-item img {
-            width: 100%; /* Makes the image responsive within the grid item */
-            height: auto;
-            max-height: auto; /* Adjust max-height to suit your design */
+          grid-template-columns: repeat(
+            3,
+            1fr
+          ); /* Creates a three-column layout */
         }
+      }
 
-        .gallery-item-description {
-            /* Add additional styling for your descriptions here */
-        }
-
-
-
-        .gallery-item a {
-            display: inline-block; /* Faz o link se ajustar ao tamanho da imagem */
-            margin-bottom: 5px; /* Espaço entre a imagem e o texto */
-        }
-
-        .gallery-item p {
-            margin: 0;
-            font-size: 1.8vw;
-            font-family: Arial, sans-serif;
-            color: #333;
+      @media (max-width: 768px) {
+        .gallery {
+          grid-template-columns: repeat(
+            2,
+            1fr
+          ); /* Two columns for smaller screens */
         }
-
-
+      }
+
+      .gallery-item {
+        display: flex;
+        flex-direction: column;
+        margin: 5px;
+        justify-content: flex-end; /* Aligns the description to the bottom */
+      }
+
+      .gallery-item img {
+        object-fit: cover; /* Ensures the image fills the parent container */
+        max-width: 100%; /* Resizes the image to fit the parent's width */
+        max-height: 100%; /* Resizes the image to fit the parent's height */
+      }
+
+      .gallery-item-description {
+        /* Add additional styling for your descriptions here */
+      }
+
+      .gallery-item a {
+        display: inline-block; /* Faz o link se ajustar ao tamanho da imagem */
+        margin-bottom: 5px; /* Espaço entre a imagem e o texto */
+      }
+
+      .gallery-item p {
+        margin: 0;
+        font-size: 1.1rem;
```

Comment:
O problema com o tamanho do texto em desktop era que estavas a usar `vw` como medida, ou seja a fonte crescia consoante o tamanho to ecrã crescia. Quando tinhamos o ecrã em desktop a font ficava demasiado grande e era cortada. É costume usar-se `rem` para fonts, e também podes usar `px`. Não sou grande expert mas neste caso deixei em rems. Depois podes mudar o tamanho